berryhill/http-response-time
# Implement requests for 5 minutes to given endpoint

This implements the given prompt of making requests to https://gitlab.com for 5 minutes. It then prints the responses times accordingly. Thereafter, prints the number of responses with that occurred within that 5 minutes

**TESTING**
+ Take note of time
+ Run application
+ Ensure requests occur synchronously
+ Ensure that 5 minutes has passed
